ugo.go

```diff
@@ -0,0 +1,223 @@
+package main
+
+import (
+	"fmt"
+	"github.com/segmentio/go-prompt"
+	"gopkg.in/alecthomas/kingpin.v2"
+	git "gopkg.in/src-d/go-git.v4"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+)
+
+const GOPKG_FILENAME = ".gopackage"
```

Comment:
The naming convention in Go is to use camelCase and PascalCase instead of underscores for pretty much everything including constants. Ideally make the file pass `golint`.